crypto/store/store_result.c

```diff
@@ -473,11 +460,14 @@ static int try_cert(struct extracted_param_data_st *data, OSSL_STORE_INFO **v,
             && (strcasecmp(data->data_type, PEM_STRING_X509_TRUSTED) == 0))
             ignore_trusted = 0;
 
+        /* Discard DER decoding errors, it only means we return "empty handed" */
+        ERR_set_mark();
```

Comment:
Again, should we clear the errors if data->object_type == OSSL_OBJECT_CERT ?

Reply:
Sure